pinot-core/src/main/java/org/apache/pinot/core/data/manager/offline/DimensionTableDataManager.java

```diff
@@ -153,6 +178,15 @@ protected void doShutdown() {
     closeDimensionTable(_dimensionTable.get());
   }
 
+  /* This method is necessary for testing because shutdown doesn't release
```

Comment:
Use proper javadoc format (starts with `/**`)

Reply:
Should we use this method in `doShutDown()`?

Reply:
Done.